ostelco-ios-client/ModelControllers/UserDefaultsWrapper.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  UserDefaultsWrapper.swift
+//  ostelco-ios-client
+//
+//  Created by Ellen Shapiro on 4/8/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import Foundation
+
+struct UserDefaultsWrapper {
+    
+    // Underlying keys for user defaults
+    private enum Key: String, CaseIterable {
+        case selectedTheme = "SelectedTheme"
+    }
+    
+    private static var defaults: UserDefaults {
+        return UserDefaults.standard
```

Comment:
This is set up as a wrapper because it's possible that to support some stuff in an extension, we may need to move this to a framework and use a shared user defaults suite.

Also it makes things cleaner at the call site.